app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    OutdatedVaccineError,
+    NotVaccinatedError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date and expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine "
+                                       "information is missing."
+                                       )
+
+        if (visitor["wearing_a_mask"] is False
+                and not visitor["wearing_a_mask"]):

```

Comment:
Use the `get` method for dictionary access can be a better approach. It's a good practice for handling cases where the presence of a key is uncertain.